pages/quickstart/getting_started.md.erb

```diff
@@ -16,6 +16,16 @@ We’ve created a simple [bash-example test repository](https://github.com/build
 
 <a class="inline-block" href="/new?template=https://github.com/buildkite/bash-example" target="__blank"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
+## Link your git email address to Buildkite
+
+Before creating a build, link the email address you have associated with your git user account to Buildkite. Until you've linked your git email address, any builds created on commits you've authored won't have the correct permissions and will fail. 
```

Comment:
This isn't true, they'll still work! And they'll be able to just hit the "Add This Email" button with the new tech that Jess has been working on… they don't need to do it ahead of time.